gc/base/GCExtensionsBase.hpp

```diff
@@ -1084,6 +1084,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		, rootScannerStatsEnabled(false)
 		, softMx(0) /* softMx only set if specified */
 		, gcThreadCountForced(false)
+		, scavengerEnabled(false)
```

Comment:
scavengerEnabled needs to stay under the OMR_GC_MODRON_SCAVENGER compile flag.  Everywhere that looks at it needs to be properly #if defined.